tests/test_subscriptions.py

```diff
@@ -227,6 +233,15 @@ def _put_subscription(self, endpoint=None, expect_code=None, attachments=None):
         )
         return resp_obj.json.get('uuid')
 
+    def _delete_subscription(self, uuid, subscription_type=None):
```

Comment:
You put [addCleanup](https://docs.python.org/3/library/unittest.html#unittest.TestCase.addCleanup) here if this function is only used for cleanup. If you do put it here, change the name of the function to `_cleanup_subscription`